### PR TITLE
Reserve 0x2 memo type for ito

### PR DESCRIPTION
**Pull request by rettetdemdativ**
[ito](https://github.com/ito-org)

### CODE REVIEW

**Review by scottleibrand — APPROVED** [review]

**Comment by rettetdemdativ** [comment]
@scottleibrand Could you also merge this? I can't do it. Or are we still waiting for approval from others?

**Pull request merged**
